canonical/ubuntu.com
# Update link on /openstack/consulting

## Done

- Updates the link for 'Download PCB datasheet' as in the [copydoc](https://assets.ubuntu.com/v1/2f580210-PCB%20Datasheet%2022.04.2024.pdf)

## QA

- Go to https://ubuntu-com-13838.demos.haus/openstack/consulting and check the link

## Issue / Card

Fixes https://docs.google.com/document/d/1f83OSIHVQGMjF7b0pZUKFvoubMGqi_b-OYbSpyhveX4/edit


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
